modules/emulator/R/GaussProcess.R

```diff
@@ -240,12 +240,12 @@ GaussProcess <- function(x, y, isotropic = TRUE, nugget = TRUE, method = "bayes"
         }
       }
       psigibbs[g, ] <- psi
-      psijump <- update(psijump, psigibbs)
+      psijump <- stats::update(psijump, psigibbs)
```

Comment:
This is actually a call to the `update` function thats within this package, not the one in stats

Reply:
Hi @mdietze, I can't find the `update` function in this package.

Reply:
https://github.com/PecanProject/pecan/blob/develop/modules/emulator/R/update.jump.R

Reply:
So this update is call to the `update.jump` function?

Reply:
well, it's either to update.jump or update.mvjump depending on whether psijump is a member of the jump class or mvjump class. This is a deliberate use of R class polymorphism https://github.com/PecanProject/pecan/blob/a2c15b9dc3748649da2c39a7e3ae22e0acee46d4/modules/emulator/R/GaussProcess.R#L139

Reply:
So all the three (line no: 218,249,264) `update` are not from `stats`

Reply:
correct